src/io/inputs/model-converter/include/antares/io/inputs/model-converter/convertorVisitor.h

```diff
@@ -62,6 +65,47 @@ class NoConstraintWithThisName final: public std::runtime_error
     }
 };
 
+class AntlrParsingError final: public std::invalid_argument
+{
+public:
+    explicit AntlrParsingError(const std::string& msg):
+        std::invalid_argument(msg)
+    {
+    }
```

Comment:
To inherit all constructors (see "delegating constructor")
```suggestion
    using std::invalid_argument::invalid_argument;
```